functional-tests/pages/additionalParams.js

```diff
@@ -0,0 +1,13 @@
+'use strict';
+
+const { BasePage } = require('kakunin');
+
+class AdditonalParamsPage extends BasePage {
+  constructor() {
+    super();
+
+    this.url = '/navigation/pages/:id/titles/:title?additionalParam1=value1&additionalParam2=value2';
```

Comment:
I would rather see query params as template too, not something that is added fully dynamically - we don't really care about concrete url, but for url matching given template.